frontend/ios/GrocerEZ/Info.plist

```diff
@@ -25,7 +25,7 @@
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
 	<key>NSLocationWhenInUseUsageDescription</key>
-	<string></string>
+	<string>Allow GrocerEZ to use your location</string>
```

Comment:
Have you actually tested this? If not, we should probably make a note to test it later when we find someone with an iPhont?

Reply:
I haven't tested it...although what is it supposed to do?